.editorconfig

```diff
@@ -7,11 +7,13 @@ indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
+[*.{java,kt}]
 # Disable wildcard imports
-[*.{java, kt}]
 ij_kotlin_name_count_to_use_star_import = 999
 ij_kotlin_name_count_to_use_star_import_for_members = 999
 ij_java_class_count_to_use_import_on_demand = 999
+# Enable trailing comma
+ktlint_disabled_rules=trailing-comma-on-call-site,trailing-comma-on-declaration-site
```

Comment:
aaaaa I didn't try this method because [the docs](https://pinterest.github.io/ktlint/0.49.0/rules/configuration-ktlint/#disabled-rules) said that `ktlint_disabled_rules` is deprecated and about to be removed … but `ktlint_standard_trailing-comma-on-call-site = disabled` etc. didn't work in my attempts.
Maybe this will change by the time `ktlint_disabled_rules` is *actually* removed 🤷

Reply:
Yup, I was stuck when following the docs, but digging around the internet showed me this alternative.

Reply:
If you want to disable them, should use
```editorconfig
ij_kotlin_allow_trailing_comma = false
ij_kotlin_allow_trailing_comma_on_call_site = false
```

Seems this PR doesn't fully enable trailing commas.